test/parallel/test-socket-write-after-fin.js

```diff
@@ -35,5 +35,6 @@ server.listen(0, common.mustCall(function() {
   sock.write('hello1');
   sock.write('hello2');
   sock.write('hello3\n');
-  sock.end('THUNDERMUSCLE!');
+  assert.strictEqual(sock.end('THUNDERMUSCLE!'), sock);
```

Comment:
since you're in here making changes already, would you mind changing the string that is written to something more innocuous here?